igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -1,32 +1,38 @@
+#include <functional>
 #include <geometry_msgs/Vector3.h>
+#include <iterator>
+#include <mutex>
```

Comment:
I don't think the `include <mutex>` here is necessary as I don't think you are using any mutexes here. I know you didn't add this, but I think since you're changing up this file it's a good idea to clean it up.